### ISSUE TITLE
[UI][API status page] The Engines list is expanding and becoming difficult to navigate effectively

### ISSUE THREAD

**Issue by Krugloff** [serpapi/public-roadmap#1908]
https://serpapi.com/status

It appears that the Engines list on the API status page is in a random order, with no option to search through it.

![image](https://github.com/user-attachments/assets/b4d6b420-47c1-452f-8b08-3e711f120ba7)

**Comment by hartator** [comment]
Yes, we need to A-Z it. There is a PR touching this: https://github.com/serpapi/SerpApi/pull/4770
